bench-tps/src/cli.rs

```diff
@@ -342,6 +344,12 @@ pub fn build_args<'a>(version: &'_ str) -> App<'a, '_> {
                 .takes_value(true)
                 .help("If set, wraps all instructions in the instruction padding program, with the given amount of padding bytes in instruction data."),
         )
+        .arg(
+            Arg::with_name("num_conflict_groups")
+                .long("num-conflict-groups")
+                .takes_value(true)
+                .help("The number of unique destination accounts per transactions 'chunk'. Lower values will result in more transaction conflicts.")
```

Comment:
maybe should add `.validate()` to check `num_conflict_groups >= 1`. (forbid 0)

Reply:
Used the existing `is_within_range`, but think this should probably be reworked to avoid the hack of `usize::MAX - 1`; Believe that should be in a separate PR, however.

Reply:
yeah, saw #29763. what a nicety. :)